### PR TITLE
FAQ and README update

### PR DESCRIPTION
**Pull request by bardkw**
Added an FAQ file and updated the README in SMQTK-Core to reference the libraries that depend on it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,22 @@
 Provide a light-weight framework for developing interfaces that have built-in
 implementation discovery and factory construction from configuration.
 
+## Libraries
+
+SMQTK-Core is used by 6 SMQTK-* libraries, all of which are pip-installable:
+
+[SMQTK-Classifier](https://github.com/Kitware/SMQTK-Classifier) provides interfaces and implementations around classification.
```
Comment (Purg) [inline README.md:11]: I might suggest a different order based on complexity:
- data provider
- image io
- descriptors
- classifier
- indexing
- relevancy

File: docs/faq.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Can I contribute to SMQTK?
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Of course! To add in your own implementation see ``CONTRIBUTING.md``. Additionally you can contribute by helping review any outstanding branches in any one of the SMQTK repos. For guidelines on reviewing please see `review_process`_
+
+.. _`review_process`: review_process.html
+
+What does SMQTK encompass?
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+SMQTK is currently composed of 7 different libraries, all of which are pip installable:
```
Comment (Purg) [inline docs/faq.rst:24]: I think we want to communicate that SMQTK[-Core] is not *just* these packages, but that this package is a "core" that these other packages are examples that utilize this, w.r.t. using the `Pluggable` and `Configurable` mixins to build out more complex functionality and provide pluggable implementations.

**Review by Purg — COMMENTED** [review]
This will need conflict resolution with current master. I would suggest putting the library section here above the "what about kwiver" section recently added.

File: docs/faq.rst (hunk 2 above)
Comment (bardkw) [reply docs/faq.rst]: I updated the answer to this question. Let me know if you think its clear enough

**Review by Purg — APPROVED** [review]

**Pull request merged**